packages/aws-cdk-lib/cx-api/lib/features.ts

```diff
@@ -1037,6 +1038,21 @@ export const FLAGS: Record<string, FlagInfo> = {
     recommendedValue: true,
   },
 
+  //////////////////////////////////////////////////////////////////////
+  [PIPELINE_REDUCE_ASSET_ROLE_TRUST_SCOPE]: {
+    type: FlagType.ApiDefault,
+    summary: 'Remove the root account principal from PipelineAssetsFileRole trust policy',
+    detailsMd: `
+      Remove the root account principal from the generated PipelineAssetsFileRole trust policy,
+      because it's only assumed by the codebuild.
+      Use a feature flag to make sure existing customers who might be relying
+      on the overly-broad trust policy are not broken.
+    `,
+    introducedIn: { v2: '2.140.0' },
```

Comment:
This should be `V2NEXT` according to the [documentation](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md#L1201) instead of explicitly specifying the version.

Reply:
Good catch. Updated.